assets/src/application/ui/layout/entityList/filterBar/EntityListFilterBar.tsx

```diff
@@ -1,16 +1,23 @@
 import * as React from 'react';
 import { AppstoreFilled, FilterOutlined, TableOutlined } from '@ant-design/icons';
 import { __ } from '@wordpress/i18n';
-import EspressoButton from '../../../input/EspressoButton';
 
+import EspressoButton from '../../../input/EspressoButton';
 import Collapsible from './Collapsible';
+import { EntityListFilterState } from './useEntityListFilterState';
 import './style.css';
 
+interface EntityListFilterBarProps {
+	entityFilters: React.ReactNode;
+	filterState: EntityListFilterState;
```

Comment:
Of course `EntityListFilterState` typing will be improved, but that isn't exactly the scope of this PR.